stock_request/models/stock_request.py

```diff
@@ -404,7 +404,7 @@ def _action_launch_procurement_rule(self):
                 )
                 self.env["procurement.group"].run(procurements)
             except UserError as error:
-                errors.append(error.name)
+                errors.append(str(error))
```

Comment:
I also saw this error when testing, but I believe we should open a new PR in order to fix this, instead of adding it here.

Reply:
Yes, sorry about that. I changed it to avoid the error but forgot to change it back.

Reply:
no problem, you can open the pr directly, hopefully we can have it fast-tracked 🙂